src/with_clause_parser.c

```diff
@@ -113,6 +114,22 @@ ts_with_clauses_parse(const List *def_elems, const WithClauseDefinition *args, S
 	return results;
 }
 
+extern TSDLLEXPORT char *
+ts_with_clause_result_deparse_value(const WithClauseResult *result)
+{
+	Oid oid = result->definition->type_id;
+	Assert(OidIsValid(oid));
+
+	Oid in_fn;
+	bool typIsVarlena;
```

Comment:
This generate errors with linters if the variable is not used.
```suggestion
	bool typIsVarlena pg_attribute_unused();
```